### PR TITLE
Add typos found in Emacs

### PR DESCRIPTION
**Pull request by skangas**
This adds some typos found in Emacs recently, excluding the ones that are too Emacs and/or Lisp specific to be suitable for general use.

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -1553,6 +1554,7 @@ alingment->alignment
 alings->aligns, slings,
 alinment->alignment
 alinments->alignments
+alis->alias
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:1564]: Perhaps`alas`, `axis`, `alms`?

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -20292,6 +20327,7 @@ keywoards->keywords
 keywork->keyword
 keyworkd->keyword
 keyworkds->keywords
+keyworks->keywords
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:20554]: Perhaps `keywork->keyword, key work,` or far-fetched?

**Comment by skangas** [comment]
I added in a bunch more typos I found just now with a separate spellchecker. I've also rebased on top of current master.

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -31364,6 +31455,7 @@ shreak->shriek
 shreshold->threshold
 shriks->shrinks
 shrinked->shrunk, shrank,
+shs->ssh
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:31458]: Could be the NHS in the UK. Not sure what to do here.

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -37236,6 +37340,7 @@ visiters->visitors
 visitng->visiting
 visivble->visible
 vissible->visible
+vist->visit
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:37343]: Perhaps `vast`, `vest`, `mist`, `list`, `fist`, `gist`, `vista`?

Also `vists->visits` (plural)

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (skangas) [reply codespell_lib/data/dictionary.txt:1564]: I like all of them, now added.

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (skangas) [reply codespell_lib/data/dictionary.txt:20554]: Hmm, yes. I think we could add it in together with `keyworks->key works`. Sounds like something you would see in a scientific paper, for example. Now done.

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (skangas) [reply codespell_lib/data/dictionary.txt:31458]: Let's suggest both. NHS makes sense.

File: codespell_lib/data/dictionary.txt (hunk 4 above)
Comment (skangas) [reply codespell_lib/data/dictionary.txt:37343]: Indeed, added. I ended up with this hunk:

```
-visted->visited
-visting->visiting
+vist->visit, vast, vest, mist, list, fist, gist, vista,
+visted->visited, listed, vested,
+visting->visiting, listing,
 vistors->visitors
+vists->visits, lists, fists,
```

**Review by DimitriPapadopoulos — APPROVED** [review]

**Comment by skangas** [comment]
Thanks for the quick merge, yet again!

**Pull request merged**